arangod/IResearch/IResearchLink.h

```diff
@@ -289,8 +288,20 @@ class IResearchLink {
     size_t indexSize{};       // size of the index in bytes
     size_t numSegments{};     // number of segments
     size_t numFiles{};        // number of files
+
+    void toPrometheus (
```

Comment:
since we plan to reuse the same stats for the upcoming InvertedIndex, I suggest we create a separate struct
```
struct LinkStats : Stats {
  void toPrometheus(
      std::string& result,
      const std::string& labels,
      const std::string& globalLabels) const;
};
```